lib/Providers/Qr/EndroidQrCodeProvider.php

```diff
@@ -34,6 +47,13 @@ public function getQRCodeImage($qrtext, $size)
         $qrCode->setBackgroundColor($this->bgcolor);
         $qrCode->setForegroundColor($this->color);
 
+        if ($this->logoPath) {
+            $qrCode->setLogoPath($this->logoPath);
+            if ($this->logoSize) {
+                $qrCode->setLogoSize($this->logoSize[0], $this->logoSize[1]);
+            }
+        }
```

Comment:
How would you suggest to add this piece of code in an extended class? Create _another_ protected method to instantiate the basic object, then use it in `getQRCodeImage()`?